Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -122,11 +122,7 @@ class BrowserCoordinator: BaseCoordinator,
         // -> delay of 0.5s to wait for animations and dismissals to finish
         if inline, !User.shared.firstTime {
             DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) { [weak self] in
-                if self?.browserViewController.presentInsightfulSheetsIfNeeded() == true {
-                    (overlayManager as? DefaultOverlayModeManager)?.overrideShouldEnterOverlayMode = false
-                } else {
-                    (overlayManager as? DefaultOverlayModeManager)?.overrideShouldEnterOverlayMode = nil
-                }
+                self?.browserViewController.presentInsightfulSheetsIfNeeded()
```

Comment:
Can we add an Ecosia comment here explaining?

Reply:
There is an Ecosia comment above this method explaining what the whole snippet does 2 lines above 👍 . I believe that's sufficient but happy to explore it further if needed.